phalcon/mvc/model/query.zep

```diff
@@ -36,6 +38,8 @@ use Phalcon\Mvc\Model\ResultsetInterface;
 use Phalcon\Mvc\Model\Resultset\Simple;
 use Phalcon\Di\InjectionAwareInterface;
 use Phalcon\Mvc\Model\RelationInterface;
+use Phalcon\Mvc\Model\TransactionInterface;
+use Phalcon\Mvc\Model as Model;
```

Comment:
1. `Model as Model` looks weird to me
2. Could you please explain where you need Model class